Makefile

```diff
@@ -0,0 +1,21 @@
+MANAGER_IMAGE ?= kubernetes-nmstate-manager
+
+all: check manager
+
+check: format vet 
+
+format:
+	gofmt -d cmd/ pkg/
```

Comment:
this won't fail if not formatted, will it?

Reply:
It will

[ellorent@quiquell kubernetes-nmstate]$ touch cmd/manager/foo.go
[ellorent@quiquell kubernetes-nmstate]$ make format 
gofmt -d cmd/ pkg/
cmd/manager/foo.go:1:1: expected 'package', found 'EOF'
make: *** [Makefile:8: format] Error 2
[ellorent@quiquell kubernetes-nmstate]$ echo $?